h/views/api_auth.py

```diff
@@ -20,6 +20,7 @@ def access_token(request):
         'access_token': token.value,
         'token_type': 'bearer',
         'expires_in': TOKEN_TTL.total_seconds(),
+        'refresh_token': token.refresh_token,
```

Comment:
I don't see this as database complexity leak out into other parts of the code. It's domain object complexity, there are tokens with and without expires / refresh tokens.

In the future we might want to whitelist specific clients that can get non-expiring tokens, as an example. This is just the inherent complexity of API tokens, and this view we're talking about here is responsible for returning any type of token that was created through the OAuth 2 token endpoint.

According to the [RFC](https://tools.ietf.org/html/rfc6749#section-5.1) it's optional for the authorization server to issue and return a refresh token, which to me means that we shouldn't render the `refresh_token` key if the token does not contain refreshing capabilities.

I would maybe even say that doing it this way makes the response generation for the token view token type agnostic, it renders the information is has. To play this even further, we might want to also only render `expires_in` when it is set, and get the time delta from `token.expires - token.created`.